File: AgentApp/Frontend/rent-ng-app/src/app/model/reservation.ts
export class Reservation {
  id: number;
  takeDate: Date;
  leaveDate: Date;
  advertId: number;

  constructor(id:number,take:Date,leave:Date,advert:number) {
    this.advertId=advert;
    this.id=id;
    this.leaveDate=leave;
    this.takeDate=take;
  }
}

File: AgentApp/Backend/src/main/java/rent/app/model/Reservation.java
package rent.app.model;

import lombok.Data;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Table
public class Reservation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date takeDate;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date leaveDate;
}
